setup.py

```diff
@@ -31,14 +31,14 @@
 ]
 
 setup(
-    name='tensornetwork',
+    name='TensorNetwork',
```

Comment:
Do not change this

Reply:
But why?

Reply:
Because it broke our pip install.

I had to fix it last time you changed it.
https://github.com/google/TensorNetwork/commit/ac0c770b2b3ac89bbab5689c3b1ac12d00f3c380

Reply:
This is one mystery worth solving.

Reply:
It's really not. Don't worry about it, we have much more important tasks to work on.